rainier-scalacheck/BUILD

```diff
@@ -10,6 +10,8 @@ scala_library(
   deps = [
     '//external:jar/com/google/code/findbugs/jsr305',
     '//external:jar/com/google/flogger/flogger',
+    '//external:jar/com/google/flogger/flogger_system_backend',
```

Comment:
Worried about this as a dep rather than a runtime dep. Might be worth moving manually.